src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -652,11 +659,15 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
     withWorkspaceContextAndPermissions(methodRepoQuery.source.workspaceName, WorkspaceAccessLevel.Read, txn) { workspaceContext =>
       withMethodConfig(workspaceContext, methodRepoQuery.source.namespace, methodRepoQuery.source.name, txn) { methodConfig =>
         import org.broadinstitute.dsde.rawls.model.MethodRepoJsonSupport._
-        RequestComplete(methodRepoDAO.postMethodConfig(
+        methodRepoDAO(context).postMethodConfig(
           methodRepoQuery.methodRepoNamespace,
           methodRepoQuery.methodRepoName,
           methodConfig.copy(namespace = methodRepoQuery.methodRepoNamespace, name = methodRepoQuery.methodRepoName),
-          userInfo))
+          userInfo) {
```

Comment:
This is what usage would look like. result is a Try so the following code could also be a match but here I wanted an exception thrown if it were a failure.